### PR TITLE
Add pyttk recipe

### PR DESCRIPTION
**Pull request by smithsp**
License type was gleaned from https://code.google.com/archive/p/python-ttk/
image

### CODE REVIEW

**Comment by smithsp** [comment]
The AppVeyor build looks like it succeeded (4 days ago), but just did not report back...

**Comment by smithsp** [comment]
The AppVeyor build seems to have completed successfully, but not reported back.

File: recipes/pyttk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    ttk
+
+about:
+  home: http://code.google.com/p/python-ttk
+  license: LGPL3
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
+  summary: Ttk Python wrapper
+  description: |
+    This ttk wrapper provides classes to allow using the Tk themed widget set, as well some extensions, under Python.
```
Comment (CJ-Wright) [inline recipes/pyttk/meta.yaml]: Could you please remove the newlines?

File: recipes/pyttk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - python
+    - tk 8.5.*
+
+test:
+  imports:
+    ttk
+
+about:
+  home: http://code.google.com/p/python-ttk
+  license: LGPL3
```
Comment (CJ-Wright) [inline recipes/pyttk/meta.yaml]: `LGPL-3.0`

File: recipes/pyttk/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: 'python setup.py install '
```
Comment (CJ-Wright) [inline recipes/pyttk/meta.yaml]: Please remove the quotes.

**Comment by smithsp** [comment]
@CJ-Wright Thanks for your help with this recipe.

**Pull request merged**